local/code/modules/surgery/mechanic_steps.dm

```diff
@@ -0,0 +1,161 @@
+//cut wires
+/datum/surgery_step/cut_wires
+	name = "cut wires"
+	implements = list(
+		TOOL_WIRECUTTER = 100,
+		TOOL_SCALPEL = 75,
+		/obj/item/knife	= 50,
+		/obj/item = 10,
+	) // 10% success with any sharp item.
+	time = 2.4 SECONDS
+	preop_sound = 'sound/items/tools/wirecutter.ogg'
+
+/datum/surgery_step/cut_wires/preop(mob/user, mob/living/carbon/target, target_zone, obj/item/tool, datum/surgery/surgery)
+	display_results(
+		user,
+		target,
+		span_notice("You begin to cut loose wires in [target]'s [parse_zone(target_zone)]..."),
+		span_notice("[user] begins to cut loose wires in [target]'s [parse_zone(target_zone)]."),
+		span_notice("[user] begins to cut loose wires in [target]'s [parse_zone(target_zone)]."),
+	)
+
+/datum/surgery_step/cut_wires/tool_check(mob/user, obj/item/tool)
+	if(implement_type == /obj/item && !tool.get_sharpness())
+		return FALSE
+	return TRUE
+
+//pry off plating
+/datum/surgery_step/pry_off_plating
+	name = "pry off plating (crowbar or hemostat)"
+	implements = list(
+		TOOL_CROWBAR = 100,
+		TOOL_HEMOSTAT = 10,
+	)
+	time = 2.4 SECONDS
+	preop_sound = 'sound/items/tools/crowbar.ogg'
+
+/datum/surgery_step/pry_off_plating/success(mob/user, mob/living/carbon/target, target_zone, obj/item/tool, datum/surgery/surgery)
+	do_sparks(rand(5, 9), FALSE, target.loc)
+	return TRUE
+
+/datum/surgery_step/pry_off_plating/preop(mob/user, mob/living/carbon/target, target_zone, obj/item/tool, datum/surgery/surgery)
+	display_results(
+		user,
+		target,
+		span_notice("You begin to pry off [target]'s [parse_zone(target_zone)] plating..."),
+		span_notice("[user] begins to pry off [target]'s [parse_zone(target_zone)] plating."),
+		span_notice("[user] begins to pry off [target]'s plating."),
+	)
+
+//weld plating
+/datum/surgery_step/weld_plating
+	name = "weld plating"
+	implements = list(
+		TOOL_WELDER = 100,
+	)
+	time = 2.4 SECONDS
+	preop_sound = 'sound/items/tools/welder.ogg'
+
+/datum/surgery_step/weld_plating/tool_check(mob/user, obj/item/tool)
+	if(implement_type == TOOL_WELDER && !tool.use_tool(user, user, 0, volume=50, amount=1))
+		return FALSE
+	return TRUE
+
+/datum/surgery_step/weld_plating/preop(mob/user, mob/living/carbon/target, target_zone, obj/item/tool, datum/surgery/surgery)
+	display_results(
+		user,
+		target,
+		span_notice("You begin to weld [target]'s [parse_zone(target_zone)] plating..."),
+		span_notice("[user] begins to weld [target]'s [parse_zone(target_zone)] plating."),
+		span_notice("[user] begins to weld [target]'s plating."),
+	)
+
+//weld plating - Act
+/datum/surgery_step/weld_plating_slice
```

Comment:
why is there two that do the same thing

Reply:
apparently just to have a diff result message. we don't need that